### PR TITLE
Removed a now unnecessary compatibility fix

### PR DESCRIPTION
**Pull request by Geoniette**
A pull request created during the git tutorial. Removes an outdated compatibility fix.

### CODE REVIEW

**Comment by tjhei** [comment]
Thank you, can you please make sure that your username and email address is set correctly? See ``git config --global -l``. Please ask one of us if you need help with that.

File: source/volume_of_fluid/reconstruct.cc [hunk 1]
```diff
@@ -154,8 +154,6 @@ namespace aspect
                         if (neighbor->level() == cell->level() &&
 #if DEAL_II_VERSION_GTE(9,2,0)
```
Comment (tjhei) [inline source/volume_of_fluid/reconstruct.cc:155]: Remove this line starting with ``#if`` as well

File: source/volume_of_fluid/reconstruct.cc [hunk 2]
```diff
@@ -154,8 +154,6 @@ namespace aspect
...
                             neighbor->is_active())
-#else
-                            neighbor->active())
 #endif
```
Comment (tjhei) [inline source/volume_of_fluid/reconstruct.cc:157]: and remove this line.

File: source/volume_of_fluid/reconstruct.cc [hunk 3]
```diff
@@ -195,8 +193,6 @@ namespace aspect
                                 if (neighbor->level() == cell->level() &&
 #if DEAL_II_VERSION_GTE(9,2,0)
```
Comment (tjhei) [inline source/volume_of_fluid/reconstruct.cc:194]: same here

File: source/volume_of_fluid/reconstruct.cc [hunk 4]
```diff
@@ -195,8 +193,6 @@ namespace aspect
...
                                     neighbor->is_active())
-#else
-                                    neighbor->active())
 #endif
```
Comment (tjhei) [inline source/volume_of_fluid/reconstruct.cc:196]: same here

File: source/volume_of_fluid/reconstruct.cc [hunk 5]
```diff
@@ -152,11 +152,9 @@ namespace aspect
                         const typename DoFHandler<dim>::cell_iterator neighbor =
                           cell->neighbor_or_periodic_neighbor(neighbor_no);
                         if (neighbor->level() == cell->level() &&
-#if DEAL_II_VERSION_GTE(9,2,0)
+
```
Comment (gassmoeller) [inline source/volume_of_fluid/reconstruct.cc:155]: Please remove this empty line

File: source/volume_of_fluid/reconstruct.cc [hunk 6]
```diff
@@ -152,11 +152,9 @@ namespace aspect
...
                             neighbor->is_active())
-#else
-                            neighbor->active())
-#endif
+
```
Comment (gassmoeller) [inline source/volume_of_fluid/reconstruct.cc:157]: and this one, too

**Comment by tjhei** [comment]
@gassmoeller the commit authorship is still not correect:
```
Author: Geoniette <41721619+Geoniette@users.noreply.github.com>
Author: Antoniette Greta Grima <agrima@IG-D02228.ig.utexas.edu>
Author: Geoniette <41721619+Geoniette@users.noreply.github.com>
``

**Comment by gassmoeller** [comment]
We will fix that today before the daily rounds

File: source/simulator/assembly.cc [hunk 7]
```diff
@@ -457,9 +454,7 @@ namespace aspect
 #ifdef ASPECT_USE_PETSC
     Amg_data.symmetric_operator = false;
 #else
-#if DEAL_II_VERSION_GTE(9,2,0)
     Amg_data.constant_modes = constant_modes;
-#else
```
Comment (gassmoeller) [inline source/simulator/assembly.cc:462]: You need to remove this whole code block up unto the #endif. Maybe you did that and it got lost in our rebase this morning. But now you can simply add this change and do git push like normal.

File: source/simulator/assembly.cc [hunk 8]
```diff
@@ -457,14 +454,9 @@ namespace aspect
...
     // To avoid a Trilinos error, only define constant modes
     // if this mpi rank owns any DoFs:
```
Comment (gassmoeller) [inline source/simulator/assembly.cc:459]: I should have been more precise: These two comment lines also need to go, they belong to the code you removed in the last commit.

**Pull request merged**